nilearn/decomposition/dict_learning.py

```diff
@@ -23,7 +23,7 @@
 
 
 def _compute_loadings(components, data):
-    ridge = Ridge(fit_intercept=None, alpha=1e-8)
+    ridge = Ridge(fit_intercept=False, alpha=1e-8)
```

Comment:
Why this change ?

Reply:
It seems to me that some intended behavior wasn't enforced. For example see Ridge API of the last stable release: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html. `fit_intercept` should be a boolean.

And checking the code, it makes sense that `None` could be passed with the same behavior as `False`
https://github.com/scikit-learn/scikit-learn/blob/f3f51f9b6/sklearn/linear_model/_ridge.py#L910